### DISCUSSION TITLE
Unified alert Handling

### DISCUSSION THREAD

**Discussion by suikast42** [grafana/grafana#68501] [Alerting]
Curretly with  ( 9.5) grafana can export alarms ( but only grafana managed alarms ) 

![image](https://github.com/grafana/synthetic-monitoring-app/assets/5046525/7b6ef18b-bf08-43a1-a8f6-80ad779ae6a3)

Frutermore if the panel is connected  with Prometheus/Mimir alarms then it does not take efect in the visialisation 
![image](https://github.com/grafana/synthetic-monitoring-app/assets/5046525/ab252159-aa66-40a6-ba50-075a9e5e68b8)

Then of couse a import behevaiur like dashboards does is missing. 

For provisinig It could be very helpful if the export fotm` /api/v1/provisioning/alert-rules` as well

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82702. Feel free to continue the discussion around this there. Thank you!
